index.js

```diff
@@ -87,15 +87,30 @@ module.exports = (number, options) => {
 		number = -number;
 	}
 
+	let localeOptions;
+
+	if (options.minimumFractionDigits !== undefined) {
+		localeOptions = {minimumFractionDigits: options.minimumFractionDigits};
+	}
+
+	if (options.maximumFractionDigits !== undefined) {
+		localeOptions = Object.assign({maximumFractionDigits: options.maximumFractionDigits}, localeOptions);
+	}
+
 	if (number < 1) {
-		const numberString = toLocaleString(number, options.locale);
+		const numberString = toLocaleString(number, options.locale, localeOptions);
```

Comment:
```suggestion
		const numberString = toLocaleString(number, options.locale, {
			...{
				options.minimumFractionDigits,
				options.maximumFractionDigits
			}
		});
```

Reply:
`localeOptions` should be `undefined` if both `minimumFractionDigits` and `maximumFractionDigits` is not set - so that it does not get localized by `toLocaleString`.